src/create-repo-relationship.ts

```diff
@@ -11,7 +41,7 @@ export async function createRepoRelationships(depsList, mainRepo, j1Client, repo
         const relationshipKey = mainRepo[0].entity._key + '|uses|' + depRepo[0].entity._key;
         const relationship = await j1Client.createRelationship(
           relationshipKey,
-          'bitbucket_repo_uses_bitbucket_repo',
+          'repo_dependency',
```

Comment:
I know a lot of our integrations do the `${from entity type}_${relationship class}_${to entity type}` for the relationship `_type` and I think thats why Matt did it that way.  Do we not want to follow that here?  I know that the `_type` is almost entirely not used.

Reply:
this is because it covers both github repos and bitbucket repos.  The relationship is not at a `_type` level.